api/data_refinery_api/views.py

```diff
@@ -845,6 +845,28 @@ class ProcessorJobList(generics.ListAPIView):
 # Statistics
 ###
 
+class AboutStats(APIView):
+    """ Returns general stats for the site, used in the about page """
+    @method_decorator(cache_page(10 * 60))
+    def get(self, request, version, format=None):
+        result = {
+            # count the total number of samples that are processed or that have a quant.sf file associated with them
+            'samples_available': Sample.objects.filter(is_processed=True).count() \
+                + Sample.objects.filter(is_processed=False, technology='RNA-SEQ', results__computedfile__filename='quant.sf').distinct().count(),
```

Comment:
Could you break this up a bit more? There's a lot going on in this one (but split into two) line of code. Maybe like init 2 variables and set `samples_available` to the sum in this line or something?